src/main/java/core/basesyntax/service/impl/ParseToListImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.model.Operation.fromString;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.ParseToList;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParseToListImpl implements ParseToList {
+    public static final int OPERATION = 0;
+    public static final int FRUIT = 1;
+    public static final int QUANTITY = 2;
+
+    @Override
+    public List<TransactionDto> parseToTransactions(List<String> dataFromFile) {
+        if (dataFromFile == null) {
+            throw new RuntimeException(String.format("No data in the file %s", dataFromFile));
+        }
+        List<TransactionDto> convertedToTransaction = new ArrayList<>();
+        for (String value : dataFromFile) {
+            String[] splittedData = value.split(",");
```

Comment:
```suggestion
            String[] splittedData = value.split(",");
```
we can use constant in this case